main-site/yarn.lock

```diff
@@ -2489,6 +2489,11 @@
   dependencies:
     "@types/node" "*"
 
+"@types/stylis@^4.0.2":
+  version "4.2.4"
+  resolved "https://registry.npmjs.org/@types/stylis/-/stylis-4.2.4.tgz"
+  integrity sha512-36ZrGJ8fgtBr6nwNnuJ9jXIj+bn/pF6UoqmrQT7+Y99+tFFeHHsoR54+194dHdyhPjgbeoNz3Qru0oRt0l6ASQ==
+
```

Comment:
Also not sure if we should be pushing up yarn lock files i forget when were allowed to

Reply:
oh idk, if so my bad